src/ngraph/runtime/interpreter/int_executable.hpp

```
@@ -436,6 +436,12 @@ class ngraph::runtime::interpreter::INTExecutable : public Executable
                                     broadcast_axes);
             break;
         }
+        case OP_TYPEID::BroadcastDistributed: {
+#ifdef NGRAPH_DISTRIBUTED_ENABLE
+            break;
+#endif
+            break;
```

We should throw an exception for unsupported ops, not silently ignore them.

---

Added support for distributed broadcast op

---

Even better  :)